scene/animation/animation_tree.h

```diff
@@ -181,6 +181,8 @@ class AnimationNode : public Resource {
 
 	void _validate_property(PropertyInfo &p_property) const;
 
+	void _notify_tree(int p_what);
```

Comment:
```suggestion
	void _animation_tree_notify(int p_what);
```
The “tree” should also be avoided as far as possible to prevent confusion between SceneTree and AnimationTree.